### DISCUSSION TITLE
Import a SetFit model (MPNetForSequenceClassification) into Spark-NLP

### DISCUSSION THREAD

**Discussion by dfustes** [JohnSnowLabs/spark-nlp#14059] [Q&A]
Hi folks! I'm trying to integrate a given classification model with Spark NLP, but I'm finding some difficulties. 

The model has been trained using [SetFit](https://github.com/huggingface/setfit). In this case, the model is built with MP-Net, plus a classification head, using Pytorch. From what I've understood from the documentation, to load this kind of model in Spark-NLP, I would need to convert it to a Tensorflow SavedModel using `TFMPNetForSequenceClassification`, and then use the corresponding annotator in Spark-NLP (I guess it would be `MPNetForSequenceClassification`, which does not exist as of today).

Could you tell if this is feasible and which are the steps needed in order to achieve this integration?

**Comment by maziyarpanahi** [comment]
Hi @dfustes,

Your steps are correct and you are also correct that the MPNet for sequence and token classifications do not exist. 

We can add them as a feature request for the next release.

**Reply by dfustes** [reply]
Hey @maziyarpanahi,

thanks a lot for your quick feedback. I would love to see that feature in Spark-NLP, so please add that feature request. I'm a Software Engineer myself, so I could contribute as well if you give me some guidance.

Cheers!

**Reply by maziyarpanahi** [reply]
https://github.com/JohnSnowLabs/spark-nlp/issues/14060
